### PR TITLE
Migrate Azure Cloud Connector Adapter from Freyja repo

### PR DESCRIPTION
**Pull request by wilyle**
Migrates the `AzureCloudConnectorAdapter` from the Freyja repo and makes doc/config updates as necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,5 +16,6 @@ Coming soon!
 
 The Freyja examples include:
 
-- [Example Freyja apps](./freyja_apps/): These are examples of how to write a Freyja application using its plugin model. Each example has a README describing its usage, and the `template` app can be used to get started with your own adapters.
-- Freyja Adapters: Coming soon!
+- [Example Freyja Apps](./freyja_apps/): These are examples of how to write a Freyja application using its plugin model. Each example will have a README describing its usage, and the `template` app can be used to get started with your own adapters.
```
Comment (ashbeitz) [inline README.md:19]: Replace "will have" with "has".

File: README.md [hunk 2]
```diff
@@ -16,5 +16,6 @@ Coming soon!
...
+- [Example Freyja Adapters](./freyja_adapters/): These are examples of adapters that can be used with Freyja. Each adapter will have a README describing its usage.
```
Comment (ashbeitz) [inline README.md:20]: Replace "will have" with "has".

**Review by ashbeitz — COMMENTED** [review]
Some minor comments.  Please take a look.

File: .github/workflows/rust-ci.yml [hunk 3]
```diff
@@ -44,17 +44,11 @@ jobs:
...
-      # The only targets currently in the repo are binary targets,
-      # so the command fails with the error message "no library targets found".
-      # This can likely be re-enabled if libraries are added to the repo.
-      # See https://github.com/rust-lang/rust/issues/50784
-      # - name: Run doctest only
-      #   uses: actions-rs/cargo@v1
-      #   with:
-      #     command: test
-      #     args: --workspace --doc
+      - name: Run doctest only
```
Comment (jorchiu) [inline .github/workflows/rust-ci.yml:47]: Can just say "Run doctest"

**Review by jorchiu — APPROVED** [review]
Added a minor comment

**Review by ashbeitz — APPROVED** [review]
Approved.

**Pull request merged**